src/styles/vendors/_antd.scss

```diff
@@ -39,3 +41,26 @@
     }
   }
 }
+
+// Grid
+.ant-layout-content {
+  padding: 0 $gap-big;
+}
+
+// Forms
+.ant-select {
+  $input-height: 50px;
+  .ant-select-selection {
+    &--single {
```

Comment:
what `--single` does ?

Reply:
it's additional class added by default from `ant design` for each ant select which is single type. 
`.ant-select-selection--single`